mne/viz/backends/renderer.py

```diff
@@ -33,6 +33,26 @@ def _get_renderer(*args, **kwargs):
     return backend._Renderer(*args, **kwargs)
 
 
+def have_3d_backend():
+    """Return the status of 3d backend capabilities.
+
+    This function does not reload the 3d backend.
+
+    Returns
+    -------
+    status: bool
+        Return True is a valid 3d backend is found and False otherwise.
+    """
+    for name in VALID_3D_BACKENDS:
+        try:
+            importlib.import_module(name=name)
+        except ImportError:
+            pass
+        else:
+            return True
```

Comment:
Why not just:
```
try:
    get_3d_backend()
except RuntimeError:
    return False
else:
    return True
```
? It's more DRY

Reply:
The backend is reloaded in `get_3d_backend` since the lazy backend loading I did in https://github.com/mne-tools/mne-python/pull/7292#issue-371545900